routes/user.playlist.js

```diff
@@ -0,0 +1,327 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../models/user');
+var Playlist = require('../models/playlist')
+var mongoose = require('mongoose');
+var nev = require('email-verification')(mongoose);
+var bcrypt = require('bcrypt-nodejs');
+var ExportSong = require('../models/exportSong')
+var Song = require('../models/song')
+var _ = require('lodash')
+var app = require('../app')
+var pdf = require('html-pdf')
+var fs = require('file-system')
+
+
+router.get('/', function(req, res, next) {
+    Playlist.find({
+            owner: req.user._id
+        })
+        .populate('songs')
+        .exec(function(err, playlists) {
+            if (err) return handleError(err)
+            res.render('playlist', {
+                playlists: playlists
+            })
+        })
+})
+
+router.post('/', function(req, res, next) {
+    var name = req.body.name
+    Playlist.findOne({
+            owner: req.user._id,
+            name: name
+        })
+        .populate('songs')
+        .exec(function(err, playlist) {
+            // console.log(JSON.stringify(playlist)
+            if (err) return handleError(err)
+                // var titles = playlist.songs.map((s) => s.title)
+                // var songs = playlist.map((s) => s.songs)
+                // console.log(songsTitle)
+            res.send({
+                songs: playlist.songs,
+                name: playlist.name
+            })
+        })
+})
+
+router.delete('/', function(req, res) {
```

Comment:
good, but why did I see earlier a router.post('/') that handled both of these? Clean up the old one?